src/components/TechnologyLogo.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+export default function TechnologyLogo({logo, name}) {
+  return (
+    <div className='group flex flex-col gap-6 items-center bg-pink-600 h-28 w-28 rounded transition-all hover:bg-pink-500 hover:h-32 hover:w-32 hover:shadow-2xl'>
```

Comment:
Check [this](https://www.google.com/url?sa=i&url=https%3A%2F%2Fweb.dev%2Frendering-performance%2F&psig=AOvVaw3Z2x-bCziEbGJLp2j6wEGg&ust=1695978105872000&source=images&cd=vfe&opi=89978449&ved=0CA8QjRxqFwoTCPiLvvb4zIEDFQAAAAAdAAAAABAH). Don't animate width and height, instead always use transform for that.